core/math/color.h

```diff
@@ -90,7 +99,7 @@ struct _NO_DISCARD_ Color {
 
 	bool is_equal_approx(const Color &p_color) const;
 
-	Color clamp(const Color &p_min = Color(0, 0, 0, 0), const Color &p_max = Color(1, 1, 1, 1)) const;
+	Color clamp(const Color &p_min = Color::TRANSPARENT_BLACK, const Color &p_max = Color(1, 1, 1, 1)) const;
```

Comment:
The latter could be `Color::WHITE`, but also I don't think it actually improves readability in this case.